### ISSUE TITLE
Packages to add for setup.py

### ISSUE THREAD

**Issue by hojaeklee** [welch-lab/pyliger#3]
Currently going through `ModuleNotFoundError` after running `pip install pyliger` and trying to execute `import pyliger` in a new Python3.8 Miniconda environment:

Modules Not Found: 
1. `numexpr`
2. `seaborn`
3. `plotnine`
4. `mygene`
5. `goatools`
6. `adjustText`

```
pip install numexpr seaborn plotnine mygene goatools adjustText
```

**Comment by eijynagai** [comment]
Still having the issue here. And after installing the packages `plotnine mygene goatools adjustText` it worked.

**Comment by theAeon** [comment]
Should be fixed as of 0.1.1.
